spec/14-classes.md

```diff
@@ -58,7 +58,15 @@ object. As such, assignment of a handle does not copy the object itself.
 
 <pre>
   <i>class-declaration:</i>
-    <i>class-modifier<sub>opt</sub></i>  class  <i>name   class-base-clause<sub>opt</sub>  class-interface-clause<sub>opt</sub></i>   {   <i>trait-use-clauses<sub>opt</sub>   class-member-declarations<sub>opt</sub></i> }
+    <i>class-modifier<sub>opt</sub></i>  class  <i>name   class-base-clause<sub>opt</sub>  class-interface-clause<sub>opt</sub></i>   {   <i>class-members<sub>opt</sub></i>   }
+    
+   <i>class-members:</i>
+    <i>class-member</i>
+    <i>class-members   class-member</i>
+    
+   <i>class-member:</i>
+    <i>trait-use-clauses</i>
+    <i>class-member-declarations</i>
```

Comment:
class-declaration uses trait-use-clauses and class-member-declarations, while trait-declaration (re)uses trait-use-clauses with trait-member-declarations. As the set of members for classes is not the exact same as that for traits, we need to keep the two separate.

Separately, while researching your question, I noted that like the previous (incorrect) class-declaration, trait-declaration also does NOT allow interspersing of trait-use-clauses and trait-member-declarations, when perhaps it should.